content/en/docs/tasks/observability/distributed-tracing/opentelemetry/grpc_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155,SC2034,SC2016
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+source "tests/util/addons.sh"
+
+_deploy_and_wait_for_addons zipkin
+_deploy_and_wait_for_addons otel-collector
+
+# @setup with OpenTelemetry Tracing Extension configured to export via gRPC
+snip_mesh_grpc_exporter
```

Comment:
may not need this if running with demo profile.

Reply:
Ah true, if using the demo profile then we don't need. But do you think that's better? I feel like it's good to have them separate and explicit so users know what is going on. Most may not know/use the demo profile in production.

Reply:
it's good to mentions it in doc, but unnecessary to do it in test.

Reply:
Fixed in [4072379](https://github.com/istio/istio.io/pull/14566/commits/40723797dedc98e5f9a07f7f147cfa0f26c62f6d)